src/types/style.rs

```diff
@@ -4,6 +4,8 @@ use std::str::FromStr;
 
 use crate::errors::Error;
 
+use super::Vec2;
```

Comment:
This is a small thing, but since there's the other change could we change this to `crate::types::Vec2`? I've typically only used `super` for test imports

Reply:
Understood!  I'll knock that out.